etc/testing/deploy-manifests/validate.sh

```diff
@@ -21,27 +21,30 @@ fi
 
 # Generate a deployment manifest for many different targets using the local
 # build of 'pachctl'
+# shellcheck disable=SC2034
 custom_args=(
 --cluster-deployment-id test
 --secure
 --dynamic-etcd-nodes 3
 --etcd-storage-class storage-class
 --namespace pachyderm
 --no-expose-docker-socket
---object-store=s3
+"--object-store=s3"
```

Comment:
Why the quotes here?

Reply:
Shellcheck didn't like it for some reason, although I should be able to replace the `=` with a space and get rid of the quotes